templates/shared/pinned_repo_cards.tmpl

```diff
@@ -0,0 +1,62 @@
+<div class="ui three stackable cards">
+	{{range .PinnedRepos}}
+		<div class="ui card">
+			<div class="content">
+				<div class="header">
+					<div class="flex-item tw-items-center">
+						<div class="flex-item-main">
+							<a class="name" href="{{.Link}}">
+								{{if or $.PageIsExplore $.PageIsProfileStarList}}{{if .Owner}}{{.Owner.Name}} / {{end}}{{end}}{{.Name}}
+							</a>
+						</div>
+						<div class="flex-item-trailing">
+							{{if .IsArchived}}
+								<span class="ui basic label">{{ctx.Locale.Tr "repo.desc.archived"}}</span>
+							{{end}}
+							{{if .IsTemplate}}
+								{{if .IsPrivate}}
+									<span class="ui basic label">{{ctx.Locale.Tr "repo.desc.private_template"}}</span>
+								{{end}}
+							{{else}}
+								{{if .IsPrivate}}
+									<span class="ui basic label">{{ctx.Locale.Tr "repo.desc.private"}}</span>
+								{{end}}
+							{{end}}
```

Comment:
Didn't we have this label sub-templated yet? If not, I suggest creating `templates/shared/repo/label.tmpl` which accepts a Repo as its only argument.

Reply:
I don't think that sub-template exists yet. At least the repo main page's header does not use any template for that part and I don't find any label template related to repos. I will create it then.